package.json

```
@@ -16,19 +16,23 @@
     "build": "rollup -c",
     "lint": "eslint src",
     "lint-fix": "eslint src --fix",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "pretest": "http-server &",
+    "test": "npx cypress run || :",
```

I don't think you need `npx` inside of npm script definitions
```suggestion
    "test": "cypress run || :",
```

---

Thanks